Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/ui/details/RecoveryCertificateDetailsFragment.kt

```diff
@@ -158,14 +148,14 @@ class RecoveryCertificateDetailsFragment : Fragment(R.layout.fragment_recovery_c
 
     private fun FragmentRecoveryCertificateDetailsBinding.onNavEvent(event: RecoveryCertificateDetailsNavigation) {
         when (event) {
-            RecoveryCertificateDetailsNavigation.Back -> goBack()
+            RecoveryCertificateDetailsNavigation.Back -> popBackStack()
```

Comment:
In VM I don't see where `Back` is triggered

Reply:
`onClose` is called when `<-` is pressed. My understanding is it that it is required to navigate `Back` when a notification lead to the details screen and the certificate is not existing, right? If yes, I don't see this is happening anywhere now. I assumed you want to get the certificate in  reliable way and if it does not exist go Back

Reply:
Ah, yes, I added it back in yesterday. It might not have been there when you reviewed but it should be in there now.